cmd/server.go

```diff
@@ -84,7 +85,7 @@ func (s *Server) securityCredentialsHandler(w http.ResponseWriter, r *http.Reque
 
 func (s *Server) roleHandler(w http.ResponseWriter, r *http.Request) {
 	remoteIP := parseRemoteAddr(r.RemoteAddr)
-	role, err := s.store.Get(remoteIP)
+	role, err := s.getRole(remoteIP)
```

Comment:
Huh, `getRole` doesn't seem to be defined anywhere. It doesn't look like it's needed.